server/index.js

```diff
@@ -1,11 +1,13 @@
 const path = require('path')
-require('dotenv').config({path: path.join(__dirname,'.env')})
+require('dotenv').config({path: path.join(__dirname, '.env')})
 const express = require('express')
 const app = express()
 const dbConnection = require('./config/db.js')(app)
 const middleware = require('./config/middleware.js')(app)
 const routes = require('./routes/router.js')(app)
 const authenthication = require('./routes/Authenthication')(app)
+const passport = require('passport')
+const Auth0Strategy = require('passport-auth0')
```

Comment:
`passport` and `auth0strategy` are both already required within the `./routes/Authentication.js` so importing them here is redundant. The instances of each of these in `index.js` are never use, whereas in the authentication page they are imported/required and set up and then used. So we shouldn't need them here to make the app work.

Reply:
For sure! I can't remember why I had those imported here, but that's definitely not necessary to run the app. We can also just require the database connection instead of assigning it to a const. Once we require the script without assigning it to a variable, it'll tell JavaScript to run the script in the backgroung. Something like this should work: 

const path = require('path')
require('dotenv').config({path: path.join(__dirname, '.env')})
const express = require('express')
const app = express()
require('./config/db.js')(app)
require('./config/middleware.js')(app)
const routes = require('./routes/router.js')(app)
const authenthication = require('./routes/Authenthication')(app)

app.use('/auth', authenthication)
app.use('/api', routes)

app.use( express.static( `${__dirname}/../front-end/build` ));

const port = 3005
app.listen(port, () =>{
  console.log(`listening on port ${port}`)
})

Reply:
Oh I really like that require method. You've tested it and it works? Thats aweseome. Lets get #55 and #56 merged into master. So we have somewhere common to start on further issues we addressed.